clang/test/Driver/openacc.c

```diff
@@ -0,0 +1,14 @@
+// RUN: %clang -S -### -fopenacc %s 2>&1 | FileCheck %s --check-prefix=CHECK-DRIVER
+// CHECK-DRIVER: "-cc1" {{.*}} "-fopenacc"
+
+// RUN: %clang -S -### -fopenacc -fexperimental-openacc-macro-override=202211 %s 2>&1 | FileCheck %s --check-prefix=CHECK-MACRO-OVERRIDE
+// RUN: %clang -S -### -fopenacc -fexperimental-openacc-macro-override 202211 %s 2>&1 | FileCheck %s --check-prefix=CHECK-MACRO-OVERRIDE
```

Comment:
We cannot, because this is a 'builtin' macro, overriding it on the command line is essentially UB.  Additionally, this clarifies that it is an experimental patch, which is less likely to be a concern when we remove it in the future.  That is, the intent is to prevent folks from coming to depend on this in the future.